src/admin/paths/admin@reports@eject_expenses.yaml

```diff
@@ -16,6 +16,8 @@ post:
           properties:
             data:
               $ref: '../../components/schemas/report.yaml#/report_action_with_expenses_ids_to_eject'
+            reason:
+              $ref: '../../components/schemas/fields.yaml#/reason'
```

Comment:
Can we add the required part for the `ADMIN` role as discussed?

Reply:
- In the admin view, if an admin tries to eject expenses from its own report it should not require any reason/comment for making the API call. Hence in this case Reason field will be required false and in the backend, we will get an empty string in the reason.

- In the admin view, if an admin tries to eject expenses from any other's report, it must require a reason/comment for making the API call. Hence in this case Reason field will be required true and in the backend, we will get the reason.

Based on these two conditions, I have not set the required condition for the reason field.